### DISCUSSION TITLE
Select commands don't work for me

### DISCUSSION THREAD

**Discussion by Eta-Beta** [moudey/Shell#262] [Q&A]
Good morning everyone, as the title suggests, the commands in select file don't work. I made several changes but I never touched this code I just translated the names of the sections, but the commands just don't work. I normally use keyboard shortcuts but I don't understand why these 3 commands don't work. The remaining commands in the "File Manager" section all work.

![Screenshot 2023-05-18 115536-B](https://github.com/moudey/Shell/assets/104278263/efa479b3-90b3-4c88-9efd-8b3f602636b7)

![Screenshot 2023-05-18 122514](https://github.com/moudey/Shell/assets/104278263/80ce39d8-2cd3-4d34-8b88-0f1a199ef783)

Does anyone know why they don't work?

**Comment by moudey** [comment]
What version of Windows are you using?

**Reply by Eta-Beta** [reply]
Windows 11 Home
Build 22000.co release.210604-1628
